qutebrowser/browser/commands.py

```diff
@@ -68,7 +71,10 @@ def _set_current_index(self, idx):
 
     def _current_index(self):
         """Convenience method to get the current widget index."""
-        return self._tabbed_browser.widget.currentIndex()
+        current_index = self._tabbed_browser.widget.currentIndex()
+        if current_index <= 0:
```

Comment:
This breaks everything on the first tab, which has index 0 - see all the failing tests on CI.